### ISSUE TITLE
Outputs functionalities

### ISSUE THREAD

**Issue by mikibonacci** [mikibonacci/aiidalab-qe-vibroscopy#1]
- multiple subpanels: this is related with the possibility to run multiple phonopy calculation in aiida-vibroscopy, not yet supported;
- dielectric properties not shown
- downloadable spectra

**Comment by AndresOrtegaGuerrero** [comment]
for the interactive raman, the user should also have the option to get the single crystal spectra from `run_single_crystal_raman_intensities` (this should let the user to specify the polarizability vector incoming,outgoing) or powder

**Comment by mikibonacci** [comment]
@AndresOrtegaGuerrero  exactly, essentially the order of the correction to the force constants, If I am correct... sorry, I had not time to check better. 

But I see that in the first case (preprocess_data, 3), you add second derivative correction, in the latter you provide third derivative... So in principle the effect (the pyhisics) is different. 

I think that this is not seen by the post process utility, so you can access both spectra, even if one of them is not accurate. 

What do you think? It is bad that we cannot have both at the same time, but maybe something can be done at the VibroWorkChain level.
